### ISSUE TITLE
[Translatable] is field "id" required in translatable class?

### ISSUE THREAD

**Issue by wojto** [KnpLabs/DoctrineBehaviors#335]
Hi, i already have database with the following structure:

```
Content:
- id
```

```
ContentTranslation:
- content_id (id from table Content)
- language ("en", "de", etc.)
- title (title in specific language)
- description (content in specific language)
```

I got error:
- field id not found
  Is the field "id" required too in this table with translation? cause i already have fields "content_id" and "language" which are primary keys (but i have to remove @ID annotation to work cause of error "Single id is not allowed on composite primary key in entity").

**Comment by kibizu** [comment]
I have the same problem, did you solve it?

**Comment by wojto** [comment]
No, i didn't find answer, so i just add this field "id" to database and set it as primary key, removing primary from previous columns (content_id, language).

**Comment by eisberg** [comment]
I solved this problem and created PR #348

**Comment by TomasVotruba** [comment]
Resolved by #348 

Impelmted in https://github.com/KnpLabs/DoctrineBehaviors/pull/442
